test/index.js

```diff
@@ -198,7 +198,15 @@ describe('with lazy loading', function() {
   });
 });
 
-describe('requiring from another directory', function() {
+describe('common functionality', function () {
+  it('throws a sensible error when not found', function () {
+    var x = gulpLoadPlugins({ config: __dirname + '/package.json' });
```

Comment:
can you just pass in a config as an object `config: {}` and then assert that x.notThere() throws, or similar, rather than loading in the package.json? Keeps the test config all in the specs.

I do need to tidy up the specs though